src/nuget/Microsoft.DotNet.BuildTools.Run.nuspec

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.Run</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft DotNet Run Tool</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=518631</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>.NET Core Framework Run Tool</summary>
+    <description>This package provides build-time support for the .NET Core Framework libraries.  You should not need to reference this package from your project.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags></tags>
+    <dependencies></dependencies>
+  </metadata>
+  <files>
+    <file src="run\run.exe" target="tools\netcoreapp1.0" />
+    <file src="run\run.runtimeconfig.json" target="tools\netcoreapp1.0" />
+    <file src="run\Newtonsoft.Json.dll" target="tools\netcoreapp1.0" />
```

Comment:
I was thinking about this as well.  We could put the tools into their own subdirectory under Tools, where the subdir name is that of the nuget package.  Thoughts?